packages/jest-config/package.json

```diff
@@ -14,9 +14,13 @@
     "./package.json": "./package.json"
   },
   "peerDependencies": {
+    "esbuild": ">=0.13.12",
```

Comment:
This should be replaced with `esbuild-register`, since that is what Jest actually needs to do the work.

Reply:
You're right, however `esbuild-register` specifies `esbuild` as a peerDependency so we'll still need the user to install `esbuild`